pyproject.toml

```diff
@@ -1,28 +1,65 @@
-[build-system]
-requires = ["setuptools", "setuptools-scm"]
-build-backend = "setuptools.build_meta"
-
 [project]
-name = "KOFFI"
-description = "Known Objects From FITS Indices"
-version = "0.1.1"
+name = "koffi"
+license = {file = "LICENSE"}
+readme = "README.md"
 authors = [
-    {name = "Max West", email = "maxwest@uw.edu"}
+    { name = "Maxine West", email = "maxwest@uw.edu" }
+]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "License :: OSI Approved :: BSD License",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
 ]
-requires-python = ">=3.7"
```

Comment:
Depending on how important support for python 3.7 is for your project, you might want to do a little testing around this. Currently we know that the template works well with python 3.8, 3.9, and 3.10. But I don't think we've tested it's functionality with 3.7

Reply:
well python 3.7 isn't available for M1 silicon macs, and epyc is down... I think I might just set the requirement to be `>=3.8` since I'm not sure if I've ever even tested for 3.7 anyway